ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/DeploymentsTable.tsx

```diff
@@ -74,28 +105,46 @@ function DeploymentsTable({
     filteredSeverities,
     showCveDetailFields,
     onClearFilters,
+    tableConfig,
 }: DeploymentsTableProps) {
     const vulnerabilityState = useVulnerabilityState();
-    const colSpan = showCveDetailFields ? 6 : 5;
+    const getVisibilityClass = generateVisibilityFor(tableConfig);
```

Comment:
Maybe an impression from scrolling back and forth for review more than following the pattern:

Compare `tableConfig` here to `columnState` argument in function definition. How about?
* `generateVisibilityForColumns` although dangling prepositions do not bother me in general
* `columnVisibilityState` in call and definition

Or feel free to suggest another alternative that aligns the above with the following related names:
* `getVisibilityClass`
* `getHiddenColumnCount`